### PR TITLE
Feature/contact

### PR DESCRIPTION
**Pull request by JessFisk**

### CODE REVIEW

File: pages/contact.html [hunk 1]
```diff
@@ -27,9 +27,12 @@
             <!-- contact form -->
             <div class="container-contact-form">
                 <form>
-                    <label>Name</label>
+                    <label id="blurb" >We've been trying to contact you about your cars extended warranty.</label>
+                    <br>
```
Comment (Taiga-KB) [inline pages/contact.html:31]: Please us flex rather than br

**Comment by JessFisk** [comment]
Hi

**Review by Taiga-KB — CHANGES_REQUESTED** [review]
See comments

File: pages/contact.html (hunk 1 above)
Comment (JessFisk) [reply pages/contact.html:31]: Budget to small for flex, using <br>

**Review by Taiga-KB — APPROVED** [review]
approved

File: pages/contact.html (hunk 1 above)
Comment (Taiga-KB) [reply pages/contact.html:31]: okay, forgot we are poor

**Pull request merged**